packages.bzl

```diff
@@ -1,7 +1,7 @@
 # Each individual package uses a placeholder for the version of Angular to ensure they're
 # all in-sync. This map is passed to each ng_package rule to stamp out the appropriate
 # version for the placeholders.
-ANGULAR_PACKAGE_VERSION = "^14.0.0 || ^15.0.0"
+ANGULAR_PACKAGE_VERSION = "^15.0.0 || ^16.0.0"
```

Comment:
```suggestion
ANGULAR_PACKAGE_VERSION = "^15.0.0-0 || ^16.0.0"
```

We want to make sure Angular v15 framework while released as `@next` can still be used